python/doc/theory/probabilistic_modeling/conditional_distributions.rst

```diff
@@ -0,0 +1,457 @@
+.. _conditional_distributions:
+
+Conditional distributions
+-------------------------
+
+The library offers some modelisation capacities on conditional distributions:
+
+- Case 1: Create a joint distribution from its conditioned margins,
+- Case 2: Create a distribution which parameters are aleatory,
+- Case 3: Condition a joint distribution by some values of its margins,
+- Case 4: Create a Bayesian posterior distribution.
+
+
+Case 1: Create a joint distribution from its conditioned margins
```

Comment:
it would be joint distribution of the conditioned margins Y|X + distribution of X
I understand that is not easy to reflect that in a single title, but currently it feels missing

Reply:
ok... $Y$ is not exactly the margin of $X$: you are right..... But this is the idea....